ds/api/base.py

```diff
@@ -11,7 +11,32 @@
 
 
 class ApiView(Resource):
+    def is_authorized(self):
+        try:
+            auth = request.headers['Authorization']
```

Comment:
im not familiar w/ how request.authorization works, so there might be a better pattern that allows us to use that